homeassistant/components/peco/sensor.py

```diff
@@ -57,17 +64,24 @@ async def async_update_data():
 
     await coordinator.async_config_entry_first_refresh()
 
-    _LOGGER.info("Setting up sensor platform")
     county = conf["county"]
-    _LOGGER.info("County: %s", county)
+
+    county_name = county.lower()[0].upper() + county.lower()[1:]
```

Comment:
`.lower().upper()`?

Reply:
County names are formatted like “BUCKS”, and to get it looking more user-friendly, I just make everything lowercase except the first letter

Reply:
Oh 🙃
https://docs.python.org/3.9/library/stdtypes.html#str.capitalize